install.sh

```diff
@@ -0,0 +1,877 @@
+#     curl ... | RKE2_TOKEN=xxx RKE2_URL=https://server-url:6443 sh -
+#
+# Environment variables:
+#   - RKE2_*
+#     Environment variables which begin with RKE2_ will be preserved for the
+#     systemd service to use. Setting RKE2_URL without explicitly setting
+#     a systemd exec command will default the command to "agent", and we
+#     enforce that RKE2_TOKEN or RKE2_CLUSTER_SECRET is also set.
+#
+#   - INSTALL_RKE2_SKIP_DOWNLOAD
+#     If set to true will not download rke2 hash or binary.
+#
+#   - INSTALL_RKE2_SYMLINK
+#     If set to 'skip' will not create symlinks, 'force' will overwrite,
+#     default will symlink if command does not exist in path.
+#
+#   - INSTALL_RKE2_SKIP_ENABLE
+#     If set to true will not enable or start rke2 service.
+#
+#   - INSTALL_RKE2_SKIP_START
+#     If set to true will not start rke2 service.
+#
+#   - INSTALL_RKE2_VERSION
+#     Version of rke2 to download from github. Will attempt to download from the
+#     stable channel if not specified.
+#
+#   - INSTALL_RKE2_BIN_DIR
+#     Directory to install rke2 binary, links, and uninstall script to, or use
+#     /usr/local/bin as the default
+#
+#   - INSTALL_RKE2_BIN_DIR_READ_ONLY
+#     If set to true will not write files to INSTALL_RKE2_BIN_DIR, forces
+#     setting INSTALL_RKE2_SKIP_DOWNLOAD=true
+#
+#   - INSTALL_RKE2_SYSTEMD_DIR
+#     Directory to install systemd service and environment files to, or use
+#     /etc/systemd/system as the default
+#
+#   - INSTALL_RKE2_EXEC or script arguments
+#     Command with flags to use for launching rke2 in the systemd service, if
+#     the command is not specified will default to "agent" if RKE2_URL is set
+#     or "server" if not. The final systemd command resolves to a combination
+#     of EXEC and script args ($@).
+#
+#     The following commands result in the same behavior:
+#       curl ... | INSTALL_RKE2_EXEC="--disable=traefik" sh -s -
+#       curl ... | INSTALL_RKE2_EXEC="server --disable=traefik" sh -s -
+#       curl ... | INSTALL_RKE2_EXEC="server" sh -s - --disable=traefik
+#       curl ... | sh -s - server --disable=traefik
+#       curl ... | sh -s - --disable=traefik
+#
+#   - INSTALL_RKE2_NAME
+#     Name of systemd service to create, will default from the rke2 exec command
+#     if not specified. If specified the name will be prefixed with 'rke2-'.
+#
+#   - INSTALL_RKE2_TYPE
+#     Type of systemd service to create, will default from the rke2 exec command
+#     if not specified.
+#
+#   - INSTALL_RKE2_SELINUX_WARN
+#     If set to true will continue if rke2-selinux policy is not found.
+#
+#   - INSTALL_RKE2_ETCD_USER
+#     Create a user 'etcd'. If this value is set, the installation
+#     will chown the etcd data-dir to this user and update the etcd
+#     pod manifest.
+
+BASE_DIR="/var/lib/rancher/rke2"
+INSTALL_PATH="/usr/local/bin"
+GITHUB_URL=https://github.com/rancher/rke2/releases
+STORAGE_URL=https://storage.googleapis.com/rke2-ci-builds
```

Comment:
There is no ci builds storage sat up for rke2 yet so this wont work properly

Reply:
I left this in, in the hopes we have that soon. I can remove.